app/api/stories/stories.controller.js

```diff
@@ -53,16 +75,10 @@ const getStoryById = async (req, res) => {
  * @param {*} res 
  */
 const addStory = async (req, res) => {
-  const { title, logo_link, description, description_preview } = req.body;
+  const { title, logo_link, description, description_preview, owner_id } = req.body;
```

Comment:
probably owner should be taken from the authenticated user or at least check with authenticated user if user not super admin

Reply:
here also you should check that owner creates only a story for himself or that user is super_admin

Reply:
Done